File: week6/mario.py
from cs50 import get_int

while True:
    size = get_int("Height: ")
    if size > 0 and size < 9:
        break

for i in range(size):
    for j in range(size - i - 1):
        print(" ", end='')
    for q in range(i + 1):
        print("#", end='')
    print("")

File: Week 1 - C/Problema 01 - mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int size;

    do
    {
        size = get_int("Size:");
    }
    while (size < 1);

    for (int i = 0; i < size; i++)
    {
        for (int j = 0; j < size - i - 1; j++)
        {
            printf(" ");
        }
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}